### PR TITLE
WaterRipple Update

### PR DESCRIPTION
**Pull request by JTeijema**
Oh C'mon this is valid YAML. I checked it. CircleCi has it in for me I'm sure

### CODE REVIEW

**Comment by JTeijema** [comment]
wat.

File: _CodingChallenges/102-2d-water-ripple.md [hunk 1]
```diff
@@ -34,6 +33,12 @@ contributions:
       name: "Marc Benzahra"
     url: "https://media.giphy.com/media/5wG3oCLNct1h0wE5ML/giphy.gif"
     source: "https://github.com/1475963/be-water-my-friend"
+  - title: "Ripple in Processing with added functions and sound"
+    author:
+      name: "Jelle Teijema"
+      url: "https://jjteijema.nl"
```
Comment (gruselhaus) [inline _CodingChallenges/102-2d-water-ripple.md]:
```suggestion
      url: "http://jjteijema.nl"
```
I'm sorry, but I do not want to bother you, but your website does not seem to support https

**Comment by gruselhaus** [comment]
@JTeijema I forgot to mention that this is an excellent work 🖤

File: _CodingChallenges/102-2d-water-ripple.md (hunk 1 above)
Comment (JTeijema) [reply _CodingChallenges/102-2d-water-ripple.md]: Hahaha yeah I host for free and the ssl costs money. Thanks tho.

Comment (gruselhaus) [reply _CodingChallenges/102-2d-water-ripple.md]: @JTeijema Reminder :D When you push your update we will be able to merge.

**Comment by gruselhaus** [comment]
@JTeijema Here we go 🎉

**Comment by JTeijema** [comment]
Ahhh thanks guys :D! @gruselhaus

**Pull request merged**